src/System.Management.Automation/commands/utility/FormatAndOutput/common/DefaultFormatters/DotNetTypes_format_ps1xml.cs

```diff
@@ -533,7 +533,6 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Diagnostics_Proc
         {
             yield return new FormatViewDefinition("process",
                 TableControl.Create()
-                    .AddHeader(Alignment.Right, label: "Handles", width: 7)
```

Comment:
Maybe we shouldn't remove things like this, but make it `#if CoreCLR ... #else ... #endif` instead. The definitions make sense in full powershell.